source/scpd/scp/QuorumSetUtils.d

```diff
@@ -22,4 +22,5 @@ bool isQuorumSetSane(ref const SCPQuorumSet qSet, bool extraChecks,
     const(char)** reason = null);
 
 // normalize the quorum set, optionally removing idToRemove
-void normalizeQSet(ref SCPQuorumSet qSet, const(NodeID)* idToRemove = null);
+void normalizeQSet(ref SCPQuorumSet qSet, const(NodeID)* idToRemove = null)
+    nothrow @nogc;
```

Comment:
There's something I don't understand, so let me ask you.
Use 'try catch' at C++ to modify it.
Or should we pack this function once more and make an exception there?

Reply:
This routine does not throw any exceptions.

Reply:
I see.